### DISCUSSION TITLE
30% wrong attempts - is it clickbait to make users update to 1.1.5?

### DISCUSSION THREAD

**Discussion by moshenskyDV** [Chia-Network/chia-blockchain#4333] [General]
So developers wrote that there will be 30% not eligible attempts from old nodes. But in code update I see just additional `if` constructions that checks this blocks. So would it means that new nodes will send 30% less amount of blocks (they won't publish to network negative amount blocks, but they still do this blocks in the same amount as earlier)? And in general reward chance for 1.1.4 and 1.1.5 will be the same?

I mean that's ok that developers want to reduce rubbish blocks in the system, but if I'm right there is acceptable to regular miner to keep plotting/farming without update and get the same reward. Am I right? If so, It looks like manipulation.

**Comment by SackofFluor** [comment]
wait what?

does that mean that plots farmed with something <1.1.5 are not working properly anymore?
wtf?

**Reply by athoszxz** [reply]
I also want to know.

**Comment by ilyakooo0** [comment]
As far as I know, the 1.1.5 update does not affect plotting in any way. 1.1.4 and 1.1.5 should produce the exact same plots.

The thing that changed is that additional transaction validation was added. As stated in the release notes, 1.1.4 allowed transactions that had negative balances to be seen by the harvester as valid, however when sending the famed block over the network, the node on the other end would see it as invalid, leading to the block being dropped, the harvester disconnected, and the harvesting (not plotting) rendered useless. 

The 1.1.5 update added the extra checks to make sure that harvesters don't attempt to farm blocks that have negative balances (are known to be invalid).
